src/Keboola/DbExtractor/Extractor/Common.php

```diff
@@ -38,18 +42,18 @@ public function createConnection($params)
         return $pdo;
     }
 
-    public function testConnection()
+    public function testConnection(): void
     {
         $this->db->query("SELECT 1");
     }
 
     /**
-     * @param array $table
+     * @param array  $table
```

Comment:
weird diff, also there should be a single space only. Probably a problem in your Code Style settings:

![image](https://user-images.githubusercontent.com/642928/39519947-1073860a-4e09-11e8-9dbb-473db1d4ece2.png)